seeds/zeroex/fantom/zeroex_fantom_api_fills_deduped_sample.csv

```diff
@@ -1,6 +1,6 @@
-fantom,2023-01-01T00:00:00Z,2023-01-01T02:13:02Z,ICE,USDC,ICE-USDC,,313.643813,7.347494171021889e+20,3.13643813e+08,313.781188990094,0xf16e81dce15b08f326220742020379b855b87df9,0x04068da6c83afcfa0e13ba15a6696662335d5b75,0xa311f7ca3eb2fb98123a807f9b8e4bbbdbdcb2ee,0x0000000253757368695377617000000000000000,0x1000000000000000000000000000000000000011,0x0bc2c2e389c7b7be1476061da617e0edf77e0d1e7cb16137a1a7d943021c8a3e,0xa311f7ca3eb2fb98123a807f9b8e4bbbdbdcb2ee,0xdef189deaef76e379df891899eb5a00a94cbc250,38,BridgeFill,true,1
-fantom,2023-01-01T00:00:00Z,2023-01-01T02:13:02Z,ICE,USDC,ICE-USDC,,393.987003,9.184367713777359e+20,3.93987003e+08,394.159569307314,0xf16e81dce15b08f326220742020379b855b87df9,0x04068da6c83afcfa0e13ba15a6696662335d5b75,0xa311f7ca3eb2fb98123a807f9b8e4bbbdbdcb2ee,0x0000000253706f6f6b7953776170000000000000,0x1000000000000000000000000000000000000011,0x0bc2c2e389c7b7be1476061da617e0edf77e0d1e7cb16137a1a7d943021c8a3e,0xa311f7ca3eb2fb98123a807f9b8e4bbbdbdcb2ee,0xdef189deaef76e379df891899eb5a00a94cbc250,30,BridgeFill,true,1
-fantom,2023-01-01T00:00:00Z,2023-01-01T02:13:02Z,ICE,USDC,ICE-USDC,,156.79855,3.6737470855109444e+20,1.5679855e+08,156.8672277649,0xf16e81dce15b08f326220742020379b855b87df9,0x04068da6c83afcfa0e13ba15a6696662335d5b75,0xa311f7ca3eb2fb98123a807f9b8e4bbbdbdcb2ee,0x0000000253706972697453776170000000000000,0x1000000000000000000000000000000000000011,0x0bc2c2e389c7b7be1476061da617e0edf77e0d1e7cb16137a1a7d943021c8a3e,0xa311f7ca3eb2fb98123a807f9b8e4bbbdbdcb2ee,0xdef189deaef76e379df891899eb5a00a94cbc250,22,BridgeFill,true,1
-fantom,2023-01-01T00:00:00Z,2023-01-01T02:13:02Z,ICE,USDC,ICE-USDC,,157.950651,3.6737470855109444e+20,1.57950651e+08,158.01983338513799,0xf16e81dce15b08f326220742020379b855b87df9,0x04068da6c83afcfa0e13ba15a6696662335d5b75,0xa311f7ca3eb2fb98123a807f9b8e4bbbdbdcb2ee,0x0000001942656574686f76656e78000000000000,0x1000000000000000000000000000000000000011,0x0bc2c2e389c7b7be1476061da617e0edf77e0d1e7cb16137a1a7d943021c8a3e,0xa311f7ca3eb2fb98123a807f9b8e4bbbdbdcb2ee,0xdef189deaef76e379df891899eb5a00a94cbc250,14,BridgeFill,true,1
\ No newline at end of file
+fantom,2023-01-01,2023-01-01 21:08,WFTM,USDC,USDC-WFTM,60.28560824406663,12.123117,6.028560824406663e+19,1.2123117e+07,12.130099915392,0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83,0x04068da6c83afcfa0e13ba15a6696662335d5b75,0x534967322b518ca7910eb019220c3a2b4ca778e4,0x0000000f53796e61707365000000000000000000,0x1000000000000000000000000000000000000011,0x09812753d268ee194af25985d4e1b859fa4d26348c22e32468e744bf639de68e,0x534967322b518ca7910eb019220c3a2b4ca778e4,0x1231deb6f5749ef6ce6943a275a1d3e7486f4eae,247,BridgeFill,true,2
```

Comment:
Event 247 is having taker token [0x82f0b8b456c1a451378467398982d4834b6829c1](https://ftmscan.com/address/0x82f0b8b456c1a451378467398982d4834b6829c1):

https://ftmscan.com/tx/0x09812753d268ee194af25985d4e1b859fa4d26348c22e32468e744bf639de68e#eventlog